src/scss/_footer.scss

```diff
@@ -0,0 +1,106 @@
+//---------------Break Points---------------//
+//50em (800px)
+//56.25em (900px)
+//90em (1440px)
+
+.footer {
+	background: url("../images/brown-wood.jpg") no-repeat center;
+	background-size: cover;
+	text-align: center;
+	padding: 4em 0; //64px 0
+}
+
+.footer__container {
+	@media(min-width: 90em) {
+		width: 90em; //1440px
+		margin: auto;
+	}
+}
+
+.footer__links {
+	width: 100%;
+
+	@media(min-width: 56.25em) {
+		width: 33%;
+		float: left;
+		text-align: left;
+	}
+}
+
+.footer__link {
+	font-family: $font-avenir-roman;
+	font-size: 1em; //16px
+	font-weight: 400;
+	color: $color-white;
+	text-decoration: none;
+	display: block;
+	margin: 0.25em auto; //4px
+
+	&:hover {
+		color: $color-blue;
+	}
+
+	@media(min-width: 56.25em) {
+		margin-left: 3em; //48px
+	}
+}
+
+.footer__text {
+	font-family: $font-avenir-roman;
+	font-size: 1em; //16px
+	font-weight: 400;
+	color: $color-white;
```

Comment:
So glad to see these color variables prefixed with `$color-`. Makes them very easy to identify and also can help with coding when you or other developers use an editor with code completion. If I just start type `$color` my editor will give me a list of all variables that start with that prefix.